frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -185,6 +185,10 @@ pods:
         {{/TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
       restore-snapshot:
         goal: FINISHED
+        configs:
```

Comment:
Did moving the config up here change anything? The PR description implies that it wasn't listed at all before, looks like the effective change is just the error checking in `cmd`